### ISSUE TITLE
Failed installation (cannot write DLLs)

### ISSUE THREAD

**Issue by ferenczy** [Yubico/yubioath-desktop#126]
I just upgraded Authenticator from the version 3.0.1 to 4.0.0. The version 3.0.1 was installed into the directory `c:\Program Files (x86)\Yubico\Yubico Authenticator` and I run the installation over that into the same directory.

The installator deleted the previous version first (the whole `Yubico Authenticator` directory) and right after that, it showed a dialog box that it cannot write to `c:\Program Files (x86)\Yubico\Qt5Core.dll`. I checked the directory `c:\Program Files (x86)\Yubico\` and there was no subdirectory `Yubico Authenticator`. I tried retry a few times and then clicked the *Ignore* button. It just showed the same dialog with an another DLL and after ignoring another one, so I canceled the installation and run it again. That time it was successful.

System: Windows 7 64bit
Yubico Authenticator version: 4.0.0

**Comment by dagheyman** [comment]
Interesting. The installer should delete the directory to ensure a fresh install, but I'm not sure why it couldn't write the dll. I tried installing 3.0.1 and then directly 4.0.0 on a Windows 7 VM, but couldn't reproduce the error.

**Comment by dagheyman** [comment]
Closing this for now since it doesn't seem reproducible.

**Comment by ferenczy** [comment]
I would just add that the same issue happened again when upgrading from 4.0.0 to 4.0.1. The (whole) installation directory is properly deleted, no new directory is created and then it fails to write the DLLs.

When I restart the installer it install the application properly.

Let me know if you would need more information for the reproduction.

**Comment by ferenczy** [comment]
It just happened again while upgrading from the version 4.1.2 to 4.1.4:

![yubioath installation error](https://preview.ibb.co/c9i8mk/yubioath_installation_error.png)

When I clicked on _Ignore_, it just showed the same error with an another file, then again another one and so on. After several more I gave up, aborted and canceled the installation and executed it again. Then the installation was successful since it was like a clean installation.
